packages/components/src/__next/context/with-next.js

```diff
@@ -31,5 +27,5 @@ export function withNext(
 		return contextConnect( WrappedComponent, namespace );
 	}
 
-	return forwardRef( CurrentComponent );
+	return CurrentComponent;
```

Comment:
We cannot automatically forward the ref of any component (for example, you cannot call forward ref on a styled component like `Text`).